bundles/org.eclipse.cdt.lsp/src/org/eclipse/cdt/lsp/util/LspUtils.java

```diff
@@ -9,7 +9,7 @@
  * Contributors:
  *   See git history
  *******************************************************************************/
-package org.eclipse.cdt.lsp;
+package org.eclipse.cdt.lsp.util;
```

Comment:
Traditional, but weak name for package and type, I'm sure we can have something better here.

Reply:
Also, next API refactor iteration